localstack/services/apigateway/helpers.py

```diff
@@ -108,12 +119,14 @@ def get_apigateway_store(account_id: str = None, region: str = None) -> ApiGatew
 
 
 class Resolver:
```

Comment:
At the time, `Resolver` looked like a ok name, not anymore, we need a better name for this openapi spec reference resolver :)

Reply:
Right, I could rename it to `OpenAPISpecificationResolver` or `OASResolver`!

Reply:
I'll also refactor at some point, like Waldemar said here: https://github.com/localstack/localstack/pull/8371#pullrequestreview-1462384708

It would be nice to be able to separate the import/export code.

We've also added a bunch a new supported features to the importer, we should try to add them to the exporter too.